data/types/upload/upload_test.go

```diff
@@ -425,7 +420,7 @@ var _ = Describe("Upload", func() {
 					errorsTest.WithPointerSourceAndMeta(structureValidator.ErrorValueNotEqualTo("invalidType", "upload"), "/type", &dataTypes.Meta{Type: "invalidType"}),
 					errorsTest.WithPointerSourceAndMeta(structureValidator.ErrorValueNotExists(), "/client/name", &dataTypes.Meta{Type: "invalidType"}),
 					errorsTest.WithPointerSourceAndMeta(structureValidator.ErrorValueNotExists(), "/client/version", &dataTypes.Meta{Type: "invalidType"}),
-					errorsTest.WithPointerSourceAndMeta(structureValidator.ErrorValueStringAsTimeNotValid("invalid", "2006-01-02T15:04:05"), "/computerTime", &dataTypes.Meta{Type: "invalidType"}),
+					//errorsTest.WithPointerSourceAndMeta(structureValidator.ErrorValueStringAsTimeNotValid("invalid", "2006-01-02T15:04:05"), "/computerTime", &dataTypes.Meta{Type: "invalidType"}),
```

Comment:
Remove this test case if it's not valid anymore

Reply:
I believe the parser is now responsible for returning errors if the format is not valid. You should probably move this test to the parser instead of removing it.

Reply:
parser already has tests for string formats, and invalid formats, added test for Time format now to cover this.